handlers.go

```diff
@@ -421,7 +422,7 @@ func (t *TricksterHandler) buildRequestContext(w http.ResponseWriter, r *http.Re
 	ctx.StepParam = ctx.RequestParams[upStep][0]
 	step, err := strconv.ParseInt(ctx.StepParam, 10, 64)
 	if err != nil {
-		return nil, fmt.Errorf("failed to parse parameter %q with value %q: %v", upStep, ctx.StepParam, err)
+		return nil, errors.Wrap(err, fmt.Sprintf("failed to parse parameter %q with value %q: %v", upStep, ctx.StepParam))
```

Comment:
You'll need to remove the `%v` here and in the instances below where you've removed `err` from the `fmt.Sprintf()` args.